script.js

```diff
@@ -0,0 +1,22 @@
+const editor = document.getElementById("editor");
```

Comment:
Your codes are so well-written #feelhappywhenIreadit

But, why you don't use any library?

Reply:
Thank you! I just thought we had to do it in vanilla JS and also people started saying using React was giving them issues. I will probably be going to translate it to React or something.